site/docs/v1/tech/migration-guide/keycloak.adoc

```diff
@@ -97,7 +97,7 @@ include::docs/v1/tech/migration-guide/_password-hash-note.adoc[]
 Here's a SQL script that will query the needed fields. Update `RealmID` with the name of the Keycloak realm to export, gathered from the <<Obtaining User Data>> step.
 
```

Comment:
```suggestion

=== Exporting From MySQL
```

Reply:
And then add both of these sections to the table of contents at the top of the file.